src/login/logind-gperf.gperf

```diff
@@ -32,6 +32,8 @@ Login.HandleLidSwitch,              config_parse_handle_action,         0, offse
 Login.HandleLidSwitchExternalPower, config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_ep)
 Login.HandleLidSwitchDocked,        config_parse_handle_action,         0, offsetof(Manager, handle_lid_switch_docked)
 Login.HandleRebootKey,              config_parse_handle_action,         0, offsetof(Manager, handle_reboot_key)
+Login.HandleRebootKeyLongPress,     config_parse_handle_action,         0, offsetof(Manager, handle_reboot_key_longpress)
```

Comment:
maybe rename the field to `handle_reboot_key_long_press`, so that there's a somewhat systematic relationship between the field name and the exposed config knob.